README_GENERATE.md

```diff
@@ -23,8 +23,22 @@ Then:
 
 ## Generate the library
 
+There is a bug with version 4.3.1 which make the following request generate a broken API package:
+
     openapi-generator generate -g python -i swagger.json --package-name pds.api_client --additional-properties=packageVersion=0.2.0
-     
+
+A fork and pull request has been made to correct that. To use the patched version, do:
+
+    git clone https://github.com/tloubrieu-jpl/openapi-generator.git
```

Comment:
I would add a directory placeholder here (something like `git clone https://github.com/tloubrieu-jpl/openapi-generator.git $HOME`), then reference that in the java call below to avoid including a reference to your actual home directory:

`java -jar $HOME/openapi-generator/modulesopenapi-generator-cli/target/openapi-generator-cli.jar ...`